Library/Include/Babylon/Console.h

```diff
@@ -29,17 +25,17 @@ namespace Babylon
         using ParentT = Napi::ObjectWrap<Console>;
         using CallbackT = std::function<void(const char*, LogLevel)>;
 
-        static Napi::ObjectReference AddConsoleToEnv(Napi::Env env, CallbackT callback)
+        static Napi::ObjectReference CreateInstance(Napi::Env env, CallbackT callback)
```

Comment:
I would recommend removing the return until we need it. Returning it possibly implies that the caller has to do something with it.